cmd/gaia/init/gentx.go

```diff
@@ -129,9 +130,20 @@ following delegation and commission default parameters:
 				return err
 			}
 
+			info, err := txBldr.Keybase().Get(name)
+			if err != nil {
+				return err
+			}
+
+			if info.GetType() == kbkeys.TypeOffline {
```

Comment:
With #3748, multisig will have their own dedicated Info type (`TypeMulti`). Will we have to check both here?

Reply:
Should we wait to merge this then? Or rebase it on your branch?

Reply:
I say we merge this and then I can rebase/update.